### DISCUSSION TITLE
How to remove 'prev' and 'next' ?

### DISCUSSION THREAD

**Discussion by nibazshab** [vuepress/core#1214] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/44338441/209657617-fdb2827c-adb2-4558-8067-500b38535b7d.png)

☝ this,I don't want to see both of them

I notice [docs/#prev](https://v2.vuepress.vuejs.org/reference/default-theme/frontmatter.html#prev) tell me how to set it to a `NavLink` object or a string, but the docs don't tell me how to remove it

**Comment by Mister-Hope** [comment, marked as answer]
https://v2.vuepress.vuejs.org/reference/default-theme/extending.html#components-replacement

**Reply by nibazshab** [reply]
Is the `PageNav.vue` ? I try to use a blank file `./components/PageNav.vue` to replacement

```ts
import { getDirname, path } from '@vuepress/utils';
const __dirname = getDirname(import.meta.url);

alias: {
  '@theme/PageNav.vue': path.resolve(__dirname, './components/PageNav.vue'),
},
```

but then it complain to me `SyntaxError: At least one <template> or <script> is required in a single file component.`

**Reply by Mister-Hope** [reply]
A blank vue component should be something like:

js version:

```js
export default () => null
```

vue:

```vue
<template></template>
```

---

I am not a bot, and I AM expecting you to do some related search before asking help. I am happy to reply anything related to vuepress, but I pretty do not want to teach you basic contents before vuepress like html, js, css, vue and so on.

**Reply by Mister-Hope** [reply]
Besides, if you are unfamilar with frontend and are a noob here, you can always try to hide them with css, that's much more easier and not hard to come up with as long as you built websites and customized them before.

**Reply by nibazshab** [reply]
Thank for your help, I'm really unfamilar with frontend, in my mind a blank file should have nothing, i never think it maybe have something ,so I didn't try to search about a blank vue component
